natgeo/mortar
# Feature/mortar 91

https://jira.nationalgeographic.com/browse/MORTAR-91

Testing Instructions:

- Git checkout
- Gulp serve 
- Visit http://localhost:3000/layout/ and make sure grid examples are responsive
